diktat-rules/src/test/resources/test/chapter6/compact_initialization/ApplyWithValueArgumentExpected.kt

```diff
@@ -9,4 +9,12 @@ httpClient.doRequest()
 
 fun setDefaultUrl(httpClient: HttpClient) {
     httpClient.url = "http://example.com"
+}
+
+fun foo() {
+    val diktatExtension = project.extensions.create(DIKTAT_EXTENSION, DiktatExtension::class.java).apply {
+    inputs = project.fileTree("src").apply {
+        include("**/*.kt")
+    }
+    reporter = PlainReporter(System.out)}
```

Comment:
Why is `}` still on the same line? It should be fixed by `NewlinesRule`, is everything ok with the AST here?

Reply:
AST is fine. It is a bug in NewlinesRule.

Reply:
Is there any issue on this? If not, please create one and describe.

Reply:
Tsay, wtf,  you sound like a pussy: "oh, it is not my bug"
Create an issue or fix it by yourself